### DISCUSSION TITLE
Error When Update V5.44.0

### DISCUSSION THREAD

**Discussion by ObaPaguei** [rsuite/rsuite#3454] [General]
I'm trying to update the version of my component from 4.8.6 to 5.44.0, but when I install it, several errors start to appear in the build. What is the correct way to do this update? Do I need to use the stack component.

**Comment by simonguo** [comment]
Hi @ObaPaguei  there are breaking changes when upgrading from v4 to v5. Refer to [Migrating from v4 to v5](https://rsuitejs.com/guide/v5-features/#migration-from-v-4-to-v-5). Hope it helps you.
